src/modules/twap/containers/TwapFormWidget/index.tsx

```diff
@@ -109,6 +113,11 @@ export function TwapFormWidget() {
         </TradeTextBox>
       </styledEl.DeadlineRow>
 
+      <div>
```

Comment:
Would be better to move it to a separate container: `<TwapFormWarnings />`

Reply:
I also would like to display banners in this component, for example:
<img width="332" alt="image" src="https://github.com/cowprotocol/cowswap/assets/7122625/faba9bac-5257-4b4d-b752-2ab9e75d8ec8">

p.s. I'm working on it right now

Reply:
Also, `getTwapFormState` should be renamed to `getTwapFormValidation` and `TwapFormState` => `TwapFormValidation`. But let's do it after merging our PR's :) Just to reduce friction

Reply:
Hmmm ok.
I'll wait for your PR to update this one then